shared-libs/transitions/src/transitions/create_user_for_contacts.js

```diff
@@ -148,26 +165,31 @@ module.exports = {
       );
     }
   },
-  filter: (doc) => {
+  filter: (doc, info = {}) => {
     const contactType = contactTypeUtils.getContactType(config.getAll(), doc);
     if (
       !contactType
       || !contactTypeUtils.isPersonType(contactType)
       || !doc.user_for_contact
-      || !doc.user_for_contact.replace
     ) {
       return false;
     }
 
-    return !!Object
-      .values(doc.user_for_contact.replace)
-      .find(({ status }) => status === USER_CREATION_STATUS.READY);
+    return Boolean(isCreatingUser(doc, info) || isReplacingUser(doc));
```

Comment:
`isReplacingUser` can return `undefined` which is falsey but not `false` and I wanted to make sure the function returns a boolean mainly because one of the tests was complaining it was receiving `undefined` instead of `false`